### DISCUSSION TITLE
XLD Setting and Ripping Guide

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#36] [Announcements]
# XLD Setting and Ripping Guide

On the macOS platform, XLD is also an excellent DAE software, just like EAC. This article will introduce you to how to use it in detail.

https://zexwoo.blog/en/posts/tutorials/xld-ripping/

**Comment by wcy-magician** [comment]
感谢分享 以及这个教程也被reddit网友推荐了https://www.reddit.com/r/Soulseek/comments/1as1bab/external_cd_drive_for_ripping_and_software/

**Comment by cylissama** [comment]
Great explanation thank you so much! I have a big CD library I am ripping including Tuts Yam so I am glad to get it off the disc.

**Reply by ZexWoo** [reply]
Glad to help you. Cheers!
